client/stylesheets/_toCat.scss

```diff
@@ -4265,6 +4266,7 @@ On click the images will be displayed at normal size to complete the effect
 .profileCardImage img {
     width: 100%; 
     height: 150px;
+    border-radius: 4px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces